src/condor_ce_scitoken_exchange

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/python3
+
+import argparse
+import base64
+import json
+import os
+import subprocess
+import sys
+import time
+
+os.environ.setdefault("CONDOR_CONFIG", "/etc/condor-ce/condor_config")
+
+import htcondor
+
+
+DEFAULT_COLLECTOR_PORT="9619"
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Exchange a SciToken for a HTCondor token at a remote CE")
+    parser.add_argument("-pool", help="Remote collector to query", metavar="coll",
+        default="collector.opensciencegrid.org")
+    parser.add_argument("-name", help="CE name to exchange SciToken with", metavar="name",
+        required=True)
+    parser.add_argument("-scitoken", help="File containing SciToken to exchange", metavar="token_file",
+        required=True)
+    parser.add_argument("-debug", action="store_true", help="Enable debug mode")
+    return parser.parse_args()
+
+
+def acquire_ce_token(pool, name, token_dir, debug=False):
+    fetch_env = dict(os.environ)
+    fetch_env.setdefault('_condor_SEC_TOKEN_DIRECTORY', token_dir)
+    args = ['condor_token_fetch', '-pool', pool, '-name', name,
+        "-lifetime", "3600", '-remote', '-authz', 'READ']
+    if debug:
+        args.append('-debug')
+    result = subprocess.run(args, stderr=subprocess.PIPE,
+        stdout=subprocess.PIPE, env=fetch_env)
+    if result.returncode:
+        print("condor_token_fetch failed with error %d:" % result.returncode)
+        print(result.stderr.decode('ascii'), file=sys.stderr, end='')
```

Comment:
Is the result of condor_token_fetch guaranteed to only contain ASCII characters?

Reply:
Yup!  It's guaranteed to be base64-encoded.  I'm OK with this code throwing a nasty exception in the case of a hacked-up remote condor daemon.